doc/classes/OS.xml

```diff
@@ -613,9 +613,10 @@
 			<return type="bool">
 			</return>
 			<description>
-				Returns [code]true[/code] if the build is a debug build.
+				Returns [code]true[/code] if the Godot binary used to run the project is a [i]debug[/i] export template.
```

Comment:
That could also be the `tools=yes` build, no?

Reply:
Indeed, but that should be clear with the line below.

Reply:
Ah, yes. I'd put both on the same sentence, I didn't expect several "Returns true".
```
Returns [code]true[/code] if the Godot binary used to run the project is a [i]debug[/i] export template or if running in the editor.
```